[2025-10-17T08:05:02.558+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-17T08:05:02.574+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:04:00+00:00 [queued]>
[2025-10-17T08:05:02.579+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:04:00+00:00 [queued]>
[2025-10-17T08:05:02.580+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-17T08:05:02.592+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-10-17 08:04:00+00:00
[2025-10-17T08:05:02.598+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=251) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-17T08:05:02.599+0000] {standard_task_runner.py:63} INFO - Started process 255 to run task
[2025-10-17T08:05:02.599+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-10-17T08:04:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmp4l_5vf83']
[2025-10-17T08:05:02.602+0000] {standard_task_runner.py:91} INFO - Job 54: Subtask task_pwd
[2025-10-17T08:05:02.614+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-17T08:05:02.651+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:04:00+00:00 [running]> on host 9565199fb8e8
[2025-10-17T08:05:02.721+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T08:04:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-17T08:04:00+00:00'
[2025-10-17T08:05:02.723+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-17T08:05:02.723+0000] {logging_mixin.py:188} INFO - 1
[2025-10-17T08:05:02.724+0000] {logging_mixin.py:188} INFO - 2
[2025-10-17T08:05:02.724+0000] {logging_mixin.py:188} INFO - 3
[2025-10-17T08:05:02.724+0000] {logging_mixin.py:188} INFO - 4
[2025-10-17T08:05:02.724+0000] {logging_mixin.py:188} INFO - 5
[2025-10-17T08:05:02.725+0000] {logging_mixin.py:188} INFO - 6
[2025-10-17T08:05:02.725+0000] {logging_mixin.py:188} INFO - 7
[2025-10-17T08:05:02.725+0000] {logging_mixin.py:188} INFO - 8
[2025-10-17T08:05:02.725+0000] {logging_mixin.py:188} INFO - 9
[2025-10-17T08:05:02.725+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-17T08:05:02.726+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-17T08:05:02.732+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-10-17T08:04:00+00:00, execution_date=20251017T080400, start_date=20251017T080502, end_date=20251017T080502
[2025-10-17T08:05:02.773+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-17T08:05:02.784+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-17T08:05:02.786+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
